CSS/style.css

```diff
@@ -186,3 +186,495 @@ li a {
   justify-content: space-around;
   padding-left: 20px;
 }
+
+.works {
+  display: grid;
+  width: 405px;
+  margin-top: 24px;
+}
+
+.recent-work-title {
+  position: static;
+  height: 50px;
+  margin-left: 24px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 30px;
+  display: flex;
+  align-items: center;
+  color: black;
+  flex: none;
+}
+
+#recent-line {
+  margin-top: 180px;
+  margin-left: -354px;
+  display: flex;
+  order: 2;
+}
+
+#recent {
+  margin-top: 30px;
+  display: inline;
+}
+
+hr {
+  width: 355px;
+  border: 2px black;
+  margin-top: 40px;
+  height: 1px;
+  background: rgb(0, 0, 0);
+}
+
+#recentwork {
+  position: static;
+  width: 356px;
+  height: 237px;
+  margin-top: 180px;
+  margin-left: 24px;
+}
+
+.recent-detail {
+  margin-left: 24px;
+  width: 357px;
+  margin-right: 24px;
+}
+
+.image-recent {
+  width: 405px;
+}
+
+#multi {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 34px;
+  line-height: 44px;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.recent-detail p {
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  color: #091e42;
+  line-height: 24px;
+  font-weight: normal;
+  font-style: normal;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+#used-lg {
+  margin-left: -50px;
+  margin-top: 25px;
+}
+
+.used {
+  border: 1px solid #8993a4;
+  padding: 10px;
+  margin-left: 8px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  left: -20px;
+}
+
+.used1-fr {
+  margin-left: -30px;
+  margin-top: 25px;
+}
+
+#recent-btn {
+  margin-left: 0;
+  margin-top: 30px;
+  position: static;
+  width: 120px;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+#recent-btn:hover {
```

Comment:
Lets' not use id as a selector. IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse. It's much preferred to use classes in selectors and then apply a class to an element in the page. Additionally, IDs impact your specificity and can lead to specificity wars.